### PR TITLE
[DEPENDS ON ILS#1052]Add missing fields in the literature details

### LINKED ISSUE #376: Improve frontsite literature details page
- [x] fix the right alignment of text from right to left for responsive view
unknown

- [x] move the `License` and `Copyrights` fields to a new dedicated tab for both desktop and mobile view so that they are very well hightlighted. Check also in the Series page if anything needs to be done

### PR DESCRIPTION
**Pull request by kprzerwa**
closes #376

### CODE REVIEW

File: src/lib/modules/Document/DocumentCopyrights.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+          {licenses && (
+            <>
+              <Header as="h3">Licenses</Header>
+              {this.renderLicenses()}
+            </>
+          )}
+        </>
+      );
+    } else {
+      return 'No stored copyrights nor licenses.';
```
Comment (ntarocco) [inline src/lib/modules/Document/DocumentCopyrights.js:64]: Why `stored`?

File: src/lib/modules/Document/DocumentExtras.js [hunk 2]
```diff
@@ -61,16 +58,11 @@ export class DocumentExtras extends Component {
         </>
       );
     } else {
-      return (
-        <InfoMessage
-          header="No additional information stored."
-          content="Edit document to add additional information"
-        />
-      );
+      return 'No additional information stored.';
```
Comment (ntarocco) [inline src/lib/modules/Document/DocumentExtras.js:61]:
```suggestion
      return 'No additional information.';
```

File: src/lib/modules/Document/DocumentToc.js [hunk 3]
```diff
@@ -2,14 +2,14 @@ import PropTypes from 'prop-types';
...
 
 class DocumentToc extends Component {
   render() {
     const {
-      document: {
-        metadata: { table_of_content: tableOfContent },
-      },
+      metadata: { table_of_content: tableOfContent },
     } = this.props;
+    if (isEmpty(tableOfContent)) return 'There is no table of content';
```
Comment (ntarocco) [inline src/lib/modules/Document/DocumentToc.js:12]:
```suggestion
    if (isEmpty(tableOfContent)) return 'No table of content';
```

File: src/lib/modules/Document/DocumentToc.js [hunk 4]
```diff
@@ -2,14 +2,14 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import Overridable from 'react-overridable';
 import { List } from 'semantic-ui-react';
+import isEmpty from 'lodash/isEmpty';
```
Comment (ntarocco) [inline src/lib/modules/Document/DocumentToc.js:5]: We try to keep the `_` to remember that it is `lodash`
```suggestion
import _isEmpty from 'lodash/isEmpty';
```

File: src/lib/pages/backoffice/Document/DocumentDetails/DocumentMetadata/DocumentContents.js [hunk 5]
```diff
@@ -6,84 +6,59 @@ import { Divider, Header } from 'semantic-ui-react';
...
+        <Divider />
+        <Header as="h3">Tags</Header>
+        <LiteratureTags size="mini" tags={tags} />
+
+        <Divider />
+        <Header as="h3">Keywords</Header>
+        {!_isEmpty(keywords) ? (
+          <LiteratureKeywords keywords={keywords} />
+        ) : (
+          'There is no keywords.'
```
Comment (ntarocco) [inline src/lib/pages/backoffice/Document/DocumentDetails/DocumentMetadata/DocumentContents.js:58]:
```suggestion
          'No keywords.'
```

File: src/lib/pages/backoffice/Document/DocumentDetails/DocumentMetadata/DocumentSubjects.js [hunk 6]
```diff
@@ -28,10 +28,10 @@ export class DocumentSubjects extends Component {
       }
       return <MetadataTable rows={rows} />;
     }
-    return null;
+    return 'There is no subject classification information.';
```
Comment (ntarocco) [inline src/lib/pages/backoffice/Document/DocumentDetails/DocumentMetadata/DocumentSubjects.js:31]: Maybe just me, but I find weird that `There is no` at the beginning...
```suggestion
    return 'No subject classification information.';
```

File: src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/LicensesField.js [hunk 7]
```diff
@@ -11,6 +11,7 @@ import _pickBy from 'lodash/pickBy';
 
 export class LicensesField extends React.Component {
   serializer = (hit) => {
+    console.log('serializer custom');
```
Comment (ntarocco) [inline src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/LicensesField.js:14]:
```suggestion
```

**Review by ntarocco — APPROVED** [review]

File: src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/LicensesField.js (hunk 7 above)
Comment (kprzerwa) [reply src/lib/pages/backoffice/Document/DocumentForms/DocumentEditor/DocumentForm/components/LicensesField.js:14]: 👀 thanks!

**Pull request merged**